src/sentry/tasks/post_process.py

```diff
@@ -611,7 +611,9 @@ def get_event_raise_exception() -> Event:
         # Simplified post processing for transaction events.
         # This should eventually be completely removed and transactions
         # will not go through any post processing.
-        if is_transaction_event:
+        if is_transaction_event and not options.get(
+            "save_event_transactions.sample_transactions_in_save"
```

Comment:
does this option name reflect what is being moved out of post process? there is a bit more than just sampling right?

Reply:
we also do a send_recieve which is used by our onboarding feature to know whether or not a transaction has been sent. i can rename if you prefer

Reply:
something like `transactions.do_post_process_in_save` makes it more obvious to me at least what's going on at a high level

Reply:
updated.